adcm_aio_client/core/objects/cm.py

```diff
@@ -117,13 +113,51 @@ class ServicesNode(PaginatedChildAccessor[Cluster, Service, None]):
 
 
 class Component(InteractiveChildObject[Service]):
-    @property
-    def id(self: Self) -> int:
-        return int(self._data["id"])
 
     def get_own_path(self: Self) -> Endpoint:
         return (*self._parent.get_own_path(), "components", self.id)
 
 
 class ComponentsNode(NonPaginatedChildAccessor[Service, Component, None]):
     class_type = Component
+
+
+class HostProvidersNode(PaginatedChildAccessor): ...
+
+
+class Host(Deletable, InteractiveChildObject):
```

Comment:
`Host` is not a child of anything, it's self-sufficient: Accessible from client's root

Reply:
fixed